README.md

```diff
@@ -66,7 +66,7 @@ In addition to the complete model structure, Byte MlPerf will also add some typi
 | resnet50-v1.5 | cv | regular | tensorflow, pytorch | imagenet2012 | fp32 |
 | bert-base | nlp | regular | tensorflow, pytorch | squad-1.1 | fp32 |
 | wide&deep | rec | regular | tensorflow | criteo | fp32 |
-| videobert | mm  |popular | onnx | cifra100 | fp32 |
+| videobert | mm  |popular | onnx | cifar100 | fp32 |
```

Comment:
Correct the typo

In the byte_mlperf/model_zoo/videobert-onnx-fp32.json, we can see the data_set name is open_cifar.